lib/kubernetes/package.json

```diff
@@ -33,6 +33,6 @@
     "@types/request": "^2.48.4",
     "jest": "^26.6.3",
     "ts-jest": "^26.4.4",
-    "typescript": "^4.1.2"
+    "typescript": "4.4.2"
```

Comment:
Ah. Now I understand. 

This `typescript` should not be here at all :) This should simply use the top-level package.json dev dependecy.

Reply:
Removed this one, as well as one in packages/cli/ that appeared to also be a duplicate. Meanwhile these ones did not extend the root package so I left them as-is (with 4.4.2):
```
test/browser/package.json
test/test-remote/looker/package.json
test/test-remote/package.json
```